articles/reactive-condition.md

```diff
@@ -4,3 +4,5 @@ title: Condition
 ---
 
 ![Marble diagram](~/images/reactive-condition.svg)
+
+The nested workflow specifying the condition must return a sequence of type <xref href="System.Boolean"/>. This nested sequence may be synchronous or asynchronous with respect to notifications from the source sequence. After each element is emitted by the source sequence, the latest value from the nested sequence is checked. If the value is `true` then the element will be accepted and emitted by the result sequence. Otherwise, the element will be dropped.
```

Comment:
Include the "BehaviorSubject -> WorkflowOutput" motif to drive this idea home. Basically, you don't even need to "pass" the incoming value to be evaluated by the operator. It also drives home the idea that the condition will check the value of the behavior on each value of the incoming sequence.